test/e2e/apps/daemon_set.go

```diff
@@ -482,6 +485,223 @@ var _ = SIGDescribe("Daemon set [Serial]", func() {
 			framework.ExpectEqual(rollbackPods[pod.Name], true, fmt.Sprintf("unexpected pod %s be restarted", pod.Name))
 		}
 	})
+
+	ginkgo.It("should surge pods onto nodes when spec was updated and update strategy is RollingUpdate [Feature:DaemonSetUpdateSurge]", func() {
```

Comment:
Maybe add a note that we expect this to go to conformance after feature promotion? Sig-apps could be doing better on our conformance coverage and we're investigating. Having a reminder to update might be helpful.

Reply:
Sure